src/intrprtr.c

```diff
@@ -3972,8 +3866,6 @@ void             IntrAssertEnd3Args ( void )
 static Int InitKernel (
     StructInitInfo *    module )
 {
-    InitGlobalBag( &STATE(IntrState),  "src/intrprtr.c:IntrState"  );
-    InitGlobalBag( &STATE(StackObj),   "src/intrprtr.c:StackObj"   );
```

Comment:
`STATE(IntrState)` and `STATE(StackObj)` are gone, so we can (and indeed, must) remove the above.